test/font_test.py

```diff
@@ -876,7 +876,7 @@ def test_font_method_should_raise_exception_after_quit(self):
                 ("set_underline", (True,)),
                 ("size", ("any Text",)),
             ]
-            version = pygame.freetype.get_version()
+            skip_methods = set(("get_strikethrough", "set_strikethrough"))
```

Comment:
```suggestion
            skip_methods = {"get_strikethrough", "set_strikethrough"}
```
nitpick: can use set syntax here

Reply:
Wow, I had forgotten set literals were a thing.